docs/querying/sql-functions.md

```diff
@@ -1390,11 +1518,30 @@ Reverses the given expression.
 
 ## RIGHT
 
-`RIGHT(expr, [length])`
+Returns the `N` rightmost characters of an expression, where `N` is an integer value.
 
-**Function type:** [Scalar, string](sql-scalar.md#string-functions)
+* **Syntax:** `RIGHT(expr, N)`
+* **Function type:** Scalar, string 
+
+<details><summary>Example</summary>
+
+  The following example returns the `3` rightmost characters of the expression `ABCDEFG`.
 
-Returns the rightmost number of characters from an expression.
+  ```sql
+  SELECT
+    'ABCDEFG' AS "expression",
+    RIGHT('ABCDEFG', 3) AS "rightmost_characters"
+  ```
```

Comment:
```sql
SELECT
'ABCDEFG' AS "expression",
RIGHT('ABCDEFG', 3) AS "rightmost_characters"
  ```
nit: Remove the spare spaces at the beginning of the line. Check the whole topic for it. I think line 1528 has it too.

Reply:
removed leading spaces from any function I could find